wsd/COOLWSD.cpp

```diff
@@ -2081,12 +2082,12 @@ void COOLWSD::innerInitialize(Application& self)
         { "mount_jail_tree", "true" },
         { "mount_namespaces", "true" },
         { "net.connection_timeout_secs", "30" },
+        { "net.content_security_policy", "" },
+        { "net.frame_ancestors", "" },
         { "net.listen", "any" },
         { "net.proto", "all" },
-        { "net.service_root", "" },
         { "net.proxy_prefix", "false" },
-        { "net.content_security_policy", "" },
-        { "net.frame_ancestors", "" },
+        { "net.service_root", "" },
```

Comment:
seeing as these end up not changed, maybe drop this from this pr and put into a separate one (even though ordering them alphabetically looks a good idea)

Reply:
yeah, can drop it. reordered when added the other items - now .. either way is OK.